src/backend/marsha/settings.py

```diff
@@ -428,9 +428,9 @@ class Base(Configuration):
 
     # Cors
     CORS_ALLOW_ALL_ORIGINS = values.BooleanValue(False)
-    CORS_ALLOWED_ORIGINS = values.ListValue([])
+    CORS_ALLOWED_ORIGINS = values.ListValue(["http://localhost:3000"])
```

Comment:
This must not be done here. You should do this in the `Development` class instead